components/common/Header.tsx

```diff
@@ -0,0 +1,142 @@
+import Link from "next/link";
+import styled from "styled-components";
+import Head from 'next/head'
+const Header = () => {
+  return (
+    <HeaderSection>
+      <Head>
+      <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
+      <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.bundle.min.js"></script>
```

Comment:
we dont add library like this @SohaibAnwaar we add library from node modules or for cdn we add it on layout